src/components/ListEmployees.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <h1>List Employees</h1>
+    <ModalComponent @closeModal="isOpen = false" @showModal="showModal" :isOpen="isOpen">
+      <table>
+        <thead>
+          <tr>
+            <th>Full Name</th>
+            <th>Age</th>
+            <th>Address</th>
+            <th>Date of birth</th>
+            <th>Company</th>
+            <th>Avatar</th>
+          </tr>
```

Comment:
chổ này anh thử đặt 1 biến header rồi for ra xem ntnao hi

Reply:
đã fix